### PR TITLE
Enable editor toolbar buttons only within certain tags

### PR DESCRIPTION
**Pull request by ourmaninamsterdam**

### CODE REVIEW

File: app/assets/javascripts/comfortable_mexican_sofa/admin/modules/scribe/scribe-plugin-image-inserter.js [hunk 1]
```diff
@@ -25,7 +26,10 @@ function (eventsWithPromises, rangy, rangySelectionSaveRestore, filterEvent) {
       };
 
       insertImageCommand.queryEnabled = function() {
-        return true;
+        var selection = new scribe.api.Selection();
+        return !! selection.getContaining(function (node) {
+          return insertImageCommand.validNodes.indexOf(node.nodeName) !== -1;
+        });
```
Comment (tomas-stefano) [inline app/assets/javascripts/comfortable_mexican_sofa/admin/modules/scribe/scribe-plugin-image-inserter.js]: I don't understand this code here :\

Comment (ourmaninamsterdam) [inline app/assets/javascripts/comfortable_mexican_sofa/admin/modules/scribe/scribe-plugin-image-inserter.js]: When the cursor is placed within the editor, the `queryEnabled` method is actioned. So, using Scribe's selection API, it gets the containing node (`getContaining`), then checks to see if this node is present in the `validNodes` list, and returns `true` if it was found. As pointed out by @dougdroper it is already returning a boolean so have corrected that.

Comment (tomas-stefano) [inline app/assets/javascripts/comfortable_mexican_sofa/admin/modules/scribe/scribe-plugin-image-inserter.js]: Cool! Thanks. Cheers :beer:

**Comment by tomas-stefano** [comment]
:+1:

**Comment by dougdroper** [comment]
:+1:

**Pull request merged**